src/blocks/blocks/countdown/edit.js

```diff
@@ -176,23 +178,29 @@ const Edit = ({
 	// Add `border-radius` for all the platforms
 	const borderRadius = 'linked' === attributes.borderRadiusType ? attributes.borderRadius + '%' : `${ attributes.borderRadiusTopLeft }% ${ attributes.borderRadiusTopRight }% ${ attributes.borderRadiusBottomRight }% ${ attributes.borderRadiusBottomLeft }%`;
 
-	const styles = css`
-		--backgroundColor: ${ attributes.backgroundColor };
-		--borderColor: ${ attributes.borderColor };
-		--borderRadius: ${ borderRadius };
+	const inlineStyles = cleanCSS([
+		[ '--backgroundColor', attributes.backgroundColor ],
+		[ '--borderColor', attributes.borderColor ],
+		[ '--borderRadius', attributes.borderRadius ]
+	]);
+
+	const [ cssNodeName, setCSS ] = useCSSNode([]);
+	useEffect( ()=>{
+		setCSS([
+			`.otter-countdown__display-area .otter-countdown__value {
+				color: ${ attributes.valueColor };
+			}`,
+			`.otter-countdown__display-area .otter-countdown__label {
+				color: ${ attributes.labelColor };
+			}`
+		]);
+	}, [ attributes.valueColor, attributes.labelColor ]);
```

Comment:
@HardeepAsrani, I made a hook that creates a style node like Emotion `useCSSNode`. We can use it to add CCS code that needs Pseudo-classes, like `hover'. We can also add var, but it is preferable to do it via utility function (made a simple function `cleanCCS`) and to add it inline -- is more reliable this way.